### DISCUSSION TITLE
qt@5 fails to build on Ubuntu/Pop!_OS 20.04

### DISCUSSION THREAD

**Discussion by wchoston** [Homebrew/discussions#1474] [Linux]
I have been unsuccessful at installing qt@5 for use with Maxima/gnuplot. Installation has failed both when it was started as a dependency and on its own. Updating doesn't help, nor do I have any warnings from 'brew doctor'.
The output from 'brew install qt@5' is in the file
[command_out.txt](https://github.com/Homebrew/discussions/files/6488695/command_out.txt)

I notice in line 8 of the file
"patching file qtwebengine/src/3rdparty/chromium/build/mac/find_sdk.py"
Is the reference to 'mac' relevant?

Additional information:  When I run 'brew gist-logs qt@5' I get 'Error: Not Found' after some time.

Has anyone else successfully installed qt@5?

Thanks for any help.

=============================================================
In case people are more comfortable with gists, the output of some commands related to me trying to build qt@5 is here.
https://gist.github.com/wchoston/6c583cb865acf67a56caf2708ab98ce6
Note that the last two lines show that 'brew gist-logs qt@5' only gives me an error message.

The output of 'brew config' is here:
https://gist.github.com/wchoston/10fabd2894383f8930f2b1db15094693

**Comment by carlocab** [comment]
Try `brew gist-logs qt@5` to generate more complete logs of your build. (Not that I'll be able to help you with those, but someone else might be able to.)

**Reply by wchoston** [reply]
When I run 'brew gist-logs qt@5', after a while I get the output 'Error: Not Found'. I should have mentioned this.
